pkg/common/utils/utils.go

```diff
@@ -43,6 +43,16 @@ import (
 
 const userInfoKey = siCommon.DomainYuniKorn + "user.info"
 
+var pluginMode bool
+
+func SetPluginMode(value bool) {
```

Comment:
NOTE: Moved plugin flag here (from context) so that it can be used by utility functions. This effectively makes it global instead of per-context (which is fine since it's always set on startup).